mmv1/templates/terraform/examples/cloudbuildv2_connection.tf.erb

```diff
@@ -0,0 +1,16 @@
+provider "google-beta" {}
```

Comment:
```suggestion
```

Reply:
I beleive this is added by the test framework implicitly, so we can exclude it! It's likely the source of the error right now:

```
        Error: Duplicate provider configuration
        
          on terraform_plugin_test.tf line 3:
           3: provider "google-beta" {}
        
        A default (non-aliased) provider configuration for "google-beta" was already
        given at terraform_plugin_test.tf:1,1-23. If multiple configurations are
        required, set the "alias" argument for alternative configurations.
    testing_new.go:73: Error retrieving state, there may be dangling resources: exit status 1
        
        Error: Duplicate provider configuration
        
          on terraform_plugin_test.tf line 3:
           3: provider "google-beta" {}
        
        A default (non-aliased) provider configuration for "google-beta" was already
        given at terraform_plugin_test.tf:1,1-23. If multiple configurations are
        required, set the "alias" argument for alternative configurations.
```

Reply:
Done.